test/demo/k8s.yaml

```diff
@@ -12,7 +12,7 @@ metadata:
     istio: sidecar-injector
```

Comment:
This should probably move under 'kustomize/' directory, for consistency ( it was created before we started to pre-generate profiles ). Can be separate PR.

Reply:
We could do this, but it would require to redefine the demo profile from being a set of values.yaml to kustomizations, right?

Reply:
Not really. We can provide one 'kustomization' for each profile - or just support a single profile, do we really need all the demo variants ?